scss/components/_messages.scss

```diff
@@ -50,7 +56,8 @@
 }
 
 .message--small {
-  padding: 1rem;
+  background-position: 1rem 1rem;
+  padding: 4rem 1rem 1rem 1rem;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `4rem 1rem 1rem` instead of `4rem 1rem 1rem 1rem`